homeassistant/components/cover/deconz.py

```diff
@@ -49,7 +54,10 @@ class DeconzCover(CoverDevice):
     def __init__(self, cover):
         """Set up cover and add update callback to get data from websocket."""
         self._cover = cover
-        self._features = SUPPORT_OPEN | SUPPORT_CLOSE | SUPPORT_SET_POSITION
+        self._features = SUPPORT_OPEN
+        self._features |= SUPPORT_CLOSE
+        self._features |= SUPPORT_STOP
+        self._features |= SUPPORT_SET_POSITION
```

Comment:
It's cleaner to assign the variable once. Use parenthesis if needed.

Reply:
I just followed how it is done in deconz light

Reply:
Yeah, it's my opinion, not a big deal.